### DISCUSSION TITLE
Halloween card!

### DISCUSSION THREAD

**Discussion by Mettaya** [community/community#37774] [Other Feature Feedback, Questions, & Ideas]
What's your plans? And `happy` `Halloween` `2022` y'all

> 

> [Https://mettaya.com/halloweencard]()  ❤🙏 And thank you for inviting me in in this great community. I gonna always need help u, if you want.

Ps : Happy Halloween and enjoy your day. 

Warm regards from me @Mettaya :)

**Comment by GrigorijSchleifer** [comment]
Hi Mettaya, happy Halloween to you too. Greetings from Germany ... by the way, you have a typo in your account description :)

It should be student not student 

<img width="486" alt="image" src="https://user-images.githubusercontent.com/36699154/198986359-9f5a6dc3-a53b-4452-b4db-091357be7287.png">
